corehq/apps/hqmedia/tasks.py

```diff
@@ -228,6 +228,11 @@ def create_files_for_ccz(build, build_profile_id, include_multimedia_files=True,
                 )
         if include_index_files and include_multimedia_files:
             multimedia_errors = check_ccz_multimedia_integrity(build.domain, fpath)
+            if multimedia_errors:
+                multimedia_errors.append(_(
```

Comment:
Nitpick: I'd put this message at the top, so the user sees it even if there are a lot of missing files.

Reply:
https://github.com/dimagi/commcare-hq/pull/25010/commits/90acfe040047bed8eb8398dc14bfe1709103018c